### PR TITLE
ThorVG: Sync with upstream 0.7.1

### LINKED ISSUE #57259: Update thorvg to v0.7.1
### Godot version

https://github.com/godotengine/godot/commit/9df9dc77a396bcdfe362365e9511de80c1e94fc0

### System information

Windows 11, Nvidia 3000 series

### Issue description

Update thorvg to v0.7.1.

Replace url in thorvg (linux) upgrade script to "0.7.1"

Do a visual confirmation that nothing obvious in the editor startup broke.

## Reference

https://github.com/Samsung/thorvg/releases/tag/v0.7.1

https://github.com/godotengine/godot/blob/master/thirdparty/thorvg/update-thorvg.sh

### Steps to reproduce

Launch godot to check it is the older version.

### Minimal reproduction project

Not applicable.

### PR DESCRIPTION
**Pull request by AnilBK**
Changes: https://github.com/Samsung/thorvg/releases/tag/v0.7.1

Fixes: https://github.com/godotengine/godot/issues/57259

Seems like, 7.0.1 is a maintenance patch, there isn't anything that affects Godot side of things.
So, all I did was change version to 0.7.1 and run `update-thorvg.sh`.

Nothing obvious in the editor startup broke.

### CODE REVIEW

**Review by fire — APPROVED** [review]

**Comment by fire** [comment]
There were a few bugs @qarmin added. I don't know their numbers.

**Comment by AnilBK** [comment]
> There were a few bugs @qarmin added. I don't know their numbers.

https://github.com/Samsung/thorvg/issues?q=is%3Aissue+is%3Aclosed+author%3Aqarmin

Qarmin's new bugs were closed 10-12 days ago, while v0.7.1 was released 5 days ago, so most probably they all have been fixed.

**Comment by akien-mga** [comment]
Looks good, but you should also update `thirdparty/README.md` accordingly, and remove the patches which have been upstreamed:
```diff
diff --git a/thirdparty/README.md b/thirdparty/README.md
index 6333a0fe87..dc2f174b2b 100644
--- a/thirdparty/README.md
+++ b/thirdparty/README.md
@@ -606,7 +606,7 @@ instead of `miniz.h` as an external dependency.
 ## thorvg
 
 - Upstream: https://github.com/Samsung/thorvg
-- Version: 0.7.0 (e527f565b770f0a41df821e6618ccaeea94f465e, 2021)
+- Version: 0.7.1 (d53eb2a880002cb770ace1c1ace9c5dfcfc28252, 2022)
 - License: MIT
 
 Files extracted from upstream source:
@@ -614,8 +614,6 @@ Files extracted from upstream source:
 See `thorvg/update-thorvg.sh` for extraction instructions. Set the version
 number and run the script.
 
-Patches in the `patches` directory should be re-applied after updates.
-
 
 ## vhacd
 
```

BTW @hermet, the `v0.7.1` tag is listed as not belonging to any existing branch, is that intended? https://github.com/Samsung/thorvg/commit/d53eb2a880002cb770ace1c1ace9c5dfcfc28252
That would happen if you tagged the changes on a local branch, and pushed the tag without pushing the branch itself.

**Comment by akien-mga** [comment]
@AnilBK You missed part of my diff, you should remove the patches which have been merged upstream (both of them): `git rm -rf thirdparty/thorvg/patches`, and remove the paragraph about applying them in the README:
```diff
diff --git a/thirdparty/README.md b/thirdparty/README.md
index 6333a0fe87..6ea5848db3 100644
--- a/thirdparty/README.md
+++ b/thirdparty/README.md
@@ -614,8 +614,6 @@ Files extracted from upstream source:
 See `thorvg/update-thorvg.sh` for extraction instructions. Set the version
 number and run the script.
 
-Patches in the `patches` directory should be re-applied after updates.
-
 
 ## vhacd
 
```

File: thirdparty/README.md [hunk 1]
```diff
@@ -606,15 +606,14 @@ instead of `miniz.h` as an external dependency.
...
-- Version: 0.7.0 (e527f565b770f0a41df821e6618ccaeea94f465e, 2021)
+- Version: 0.7.1 (d53eb2a880002cb770ace1c1ace9c5dfcfc28252, 2022)
 - License: MIT
 
 Files extracted from upstream source:
 
 See `thorvg/update-thorvg.sh` for extraction instructions. Set the version
 number and run the script.
 
-Patches in the `patches` directory should be re-applied after updates.
```
Comment (akien-mga) [inline thirdparty/README.md:617]: There's now 3 lines between the sections instead of 2. Please remove another line (as in my diff).

**Comment by AnilBK** [comment]
> 


Yes,done.

I mis-read earlier to apply patch. I applied path but nothing changed.
I re-read your reply and found that you mentioned to remove the patches.
So, did that now. :D

File: thirdparty/README.md (hunk 1 above)
Comment (AnilBK) [reply thirdparty/README.md:617]: Done.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**